src/cloud-cli/applications/list-apps.ts

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { createGlobalLogger } from "../../telemetry/logs";
+import { getCloudCredentials } from "../utils";
+
+export async function listApps(host: string, port: string) {
+  const logger = createGlobalLogger();
+  const userCredentials = getCloudCredentials();
+  const bearerToken = "Bearer " + userCredentials.token;
+
+  try {
+    const list = await axios.get(
+      `http://${host}:${port}/${userCredentials.userName}/application`,
+      {
+        headers: {
+          Authorization: bearerToken,
+        },
+      }
+    );
+    console.log(list.data);
```

Comment:
For now print all the records -- eventually we will make it user friendly and remove unwanted details (e.g., `FCBaseConfig`)